src/assets/stylesheets/base/_buttons.scss

```diff
@@ -0,0 +1,17 @@
+.btn {
+  @apply text-base;
+
+  height: rem(56px);
+  padding: rem(20px 18px 14px);
+
+  border-radius: rem(10px);
+
+  background-color: theme('colors.white');
```

Comment:
Should we change to using the utility classes instead?
For example:
```
background-color: theme('colors.white') = bg-white
color: theme('colors.black') = text-black
font-weight: theme('fontWeight.bold') = font-bold
border-radius: rem(10px) = you can set a value on tailwind.config.js and use it here
```

Reply:
exactly yes, thanks.
updated in ad19a1a